### ISSUE TITLE
Provision to generate the results file from command line app

### ISSUE THREAD

**Issue by mohitmundhragithub** [mlcommons/mobile_app_open#627]
Right now only the whole app generates the results.json file.
I think we should add a provision to generate the results file into the command line application as well?

I understand that we run a single case from the command line. So, if we can decide some format for writing the results of the single case.  These results from each individual case can then be collated together to create a results.json file, using some other script.

**Comment by freedomtan** [comment]
@mkankana to share the python script. Let's check if the script is enough later.

**Comment by mohitmundhragithub** [comment]
https://github.com/mlcommons/inference/tree/master/vision/classification_and_detection/tools
https://github.com/mlcommons/inference/blob/master/language/bert/accuracy-squad.py

Manasa shared these links in the chat of the meeting.

**Comment by mohitmundhragithub** [comment]
@mkankana, I checked the links above. These seems to be doing the accuracy calculations using these scripts. 
I have been able to do the accuracy computation directly using the datasets code present in "flutter/cpp/datasets".
The command line tool in "flutter/cpp/main.cpp" reports the accuracy numbers from the datasets, directly on the command line.

I had created this issue for finding a way to create a results file directly from the performance and accuracy runs. These scripts may not help here.
